Source/LinkedInWebViewController.swift

```diff
@@ -37,7 +35,7 @@ class LinkedInWebViewController: UIViewController, UIWebViewDelegate {
         self.redirectUrl = redirectUrl
```

Comment:
You can add a redirectURL getter inside the LinkedInManager, so you don't need to pass it as a parameter here.